kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -10,12 +10,15 @@
 
     @Autowired
     @Qualifier("toDoList")
-    TaskList taskList;
+    TaskList toDoList;
 
-    @Bean
-    public Board getBoard() {
-        return new Board(taskList);
-    }
+    @Autowired
```

Comment:
po co ci te adnotacje?

Reply:
Autowired? Żeby skonfigurować springa. Gdy je usunę to testy nie wykonają się poprawnie.